src/ontology/mondo-edit.obo

```diff
@@ -111948,6 +111954,7 @@ subset: gard_rare {source="GARD:4", source="MONDO:GARD"}
 subset: nord_rare {source="MONDO:NORD", source="NORD:705"}
 subset: rare
 synonym: "Acanthocheilonema perstans infection" RELATED [GARD:0000004]
+synonym: "Acanthocheilonemiasis" EXACT [NORD:705]
 synonym: "Acanthocheilonemiasis" RELATED [GARD:0000004, NORD:705]
```

Comment:
The GARD content (gard-rare-sbset.owl) does not include any information about synonyms and the NORD content includes "Acanthocheilonemiasis" as an EXACT synonym so the the RELATED synonym will be removed.

Reply:
This change should not have made it through to here.. The pre-emptive QC system must be able to catch this

Reply:
The check `sparql/qc/general/qc-related-exact-synonym.sparql` was not in the list of qc items to handle. I've added it now.